functions/PayrollProcessor.Functions/Features/Payrolls/PayrollEntity.cs

```diff
@@ -58,7 +59,7 @@ public class EmployeePayrollEntity : TableEntity
     {
         public DateTimeOffset CheckDate { get; set; }
         public Guid EmployeeId { get; set; }
-        public double GrossPayroll { get; set; }
+        public decimal GrossPayroll { get; set; }
```

Comment:
Monetary values should typically be decimal in .NET

Reply:
My old C++ habits are hard to let go of!